include/envoy/secret/secret_manager.h

```diff
@@ -44,6 +45,14 @@ class SecretManager {
   virtual CertificateValidationContextConfigProviderSharedPtr
   findStaticCertificateValidationContextProvider(const std::string& name) const PURE;
 
+  /**
+   * @param name a name of the static TrustedCaConfigProviderSharedPtr.
+   * @return the TrustedCaConfigProviderSharedPtr. Returns nullptr if the static trusted CA config
+   * is not found.
+   */
+  virtual TrustedCaConfigProviderSharedPtr
+  findStaticTrustedCaConfigProvider(const std::string& name) const PURE;
```

Comment:
We may not need to support static trusted_ca.  Yes, semantic wise, we need to support it.   since  it is one of secrets.   but I don't think it will be useful.